kolibri/core/errorreports/schemas.py

```diff
@@ -0,0 +1,54 @@
+context_frontend_schema = {
+    "type": "object",
+    "properties": {
+        "browser": {"type": "string"},
+        "component": {"type": "string", "optional": True},
+        "device": {
+            "type": "object",
+            "properties": {
+                "type": {"type": "string"},  # "desktop", "tablet", "mobile"
```

Comment:
With json schema we can ensure the device meets a predefined list of device types